### DISCUSSION TITLE
Refreshing dacha2 cache after split brain

### DISCUSSION THREAD

**Discussion by chkp-liordan** [featurehub-io/featurehub#1169] [Q&A]
we have deployment of 3 pods for dacha. we see that two pod with updated data and one is behind.
there is API to refresh that data?

**Comment by rvowles** [comment]
heya  - dacha2 is an empty cache setup, so killing the pod is the best option and it will gradually refill as it is asked for api keys.

**Comment by chkp-liordan** [comment]
just FYI, the problem was in NATS. it wasn't deployed as a cluster (by mistake). therefor dacha was not updated. after redeploy NATS as a cluster it works.
how to check Dacha using the api (https://github.com/featurehub-io/featurehub/blob/a12547231dd523b68ae3107208fb725f1eabb318/backend/dacha-api/dacha-api.yaml#L7):  
1. port-forward to each pod
2. curl http://localhost:8601/v2/environment/"<client_key>" | md5
3. check for each pod if md5 is different.
